node/impl/full/eth.go

```diff
@@ -575,23 +588,18 @@ func (a *EthModule) EthGetBlockReceiptsLimited(ctx context.Context, blockParam e
 		return nil, xerrors.Errorf("failed to load state tree: %w", err)
 	}
 
+	baseFee := ts.Blocks()[0].ParentBaseFee
```

Comment:
Same logic as above. Works because `ts` here is the "inclusion" tipset for the messages.